programs/survival/shulkerboxes.sc

```diff
@@ -83,11 +83,11 @@ __on_player_collides_with_entity(player, entity) -> if (global_vacuum == 'collis
 __item_animation(e, max_age, player) ->
 (
    particle('portal', pos(e)-[0,0.3,0], 10, 0.1, 0);
-   if (e~'age' > max_age, 
+   if (e~'age' > max_age,
       [item, count, tag] = e~'item';
       items_left = __add_item_to_vacuum_sboxes(player, item, count, tag, true);
       count_to_remove = count - items_left;
-      if (count_to_remove, modify(e, 'nbt_merge', '{Item:{Count:'+(count-count_to_remove)+'b}}'));
+      if (count_to_remove, modify(e, 'nbt_merge', '{Item:{count:'+(count-count_to_remove)+'}}'));

```

Comment:
Should Item be changed to item here?

Reply:
I just remembered that I need to answer this one. `Item` should be uppercase. I tried with lowercase first but that did not work. My guess is that `{Item:` is part of scarpet and not Minecraft, but I'm not too familiar with either one.